mods/ts/rules/defaults.yaml

```diff
@@ -69,6 +69,10 @@
 		ShowToEnemies: true
 		ZOffset: 512
 		UpgradeMinEnabledLevel: 1
+	PowerMultiplier@EMPDISABLE:
+		UpgradeTypes: empdisable
+		Modifier: 0
+		UpgradeMinEnabledLevel: 1
```

Comment:
`PowerMultiplier` has `BaseLevel` instead of `UpgradeMinEnabledLevel` and it is already `1`.
`UpgradeMinEnabledLevel` should be removed.